web/migrations/008_assign_repair_id.sql

```diff
@@ -0,0 +1,14 @@
+--find max repair_id, then + 1
+CREATE OR REPLACE PROCEDURE assign_repair_id
+AS
+RETURN number IS
+	new_repair_id := 0;
+BEGIN 
+	SELECT max(repair_id) 
+		into new_repair_id 
+		from repair_job;
+	new_repair_id := new_repair_id + 1;
+	RETURN new_repair_id;
+END;
+/
+show errors;	
```

Comment:
Fixed query, add types and remove the `/` `show errors;` at the bottom.  You can use the `/ show errors` if you're testing.

```sql
--find max repair_id, then + 1
CREATE OR REPLACE FUNCTION assign_repair_id
RETURN NUMBER
AS
    new_repair_id INTEGER := 0;
BEGIN 
    SELECT MAX(repair_id) 
      INTO new_repair_id 
      FROM repair_jobs;

    new_repair_id := new_repair_id + 1;
    RETURN new_repair_id;
END;
```